test/elastic_record/index/documents_test.rb

```diff
@@ -60,7 +60,7 @@ def test_delete_document
     index.index_document('abc', { color: 'red' })
     assert index.record_exists?('abc')
 
-    index.delete_document('abc')
+    index.delete_document('abc', routing: 'abc')
     refute index.record_exists?('abc')
```

Comment:
Confirmed that this change to L63 causes the assertion on L64 to fail on master.